baseplate/server/runtime_monitor.py

```diff
@@ -272,10 +273,18 @@ def start(server_config: Dict[str, str], application: Any, pool: Pool) -> None:
         except Exception as exc:
             logger.info("monitoring.gc.refcycle disabled: %s", exc)
 
+    statsd_metrics_client = baseplate._metrics_client
+    if not statsd_metrics_client:
+        statsd_metrics_client = metrics.Client(
+            transport=metrics.NullTransport(log_if_unconfigured=False),
+            namespace="noop",
+        )
+
     thread = threading.Thread(
         name="Server Monitoring",
         target=_report_runtime_metrics_periodically,
-        args=(application.baseplate._metrics_client, reporters),
+        args=(statsd_metrics_client, reporters),
```

Comment:
I really doubt it's high, but this explicitly means we're spending CPU cycles in production on something that we're just going to throw away.